includes/blocks/course-theme/class-prev-lesson.php

```diff
@@ -47,23 +47,29 @@ public function render( array $attributes = [] ) : string {
 			return '';
 		}
 
-		$urls = sensei_get_prev_next_lessons( $lesson_id );
+		$urls           = sensei_get_prev_next_lessons( $lesson_id );
```

Comment:
Prev/next buttons contain some duplication code. Since we plan to [refactor that](https://github.com/Automattic/sensei/issues/4449), I kept it for now.